<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_market_dialog</class>
 <widget class="QDialog" name="view_market_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Market</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="viewMarketGroupBox">
       <property name="title">
        <string>View Market: Marketname</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPlainTextEdit" name="descPlainTextEdit">
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regFeeLabel">
            <property name="text">
             <string>Registration Fee: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="burnMultLabel">
            <property name="text">
             <string>BURN Multiplier: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="downvoteMultLabel">
            <property name="text">
             <string>Downvote Multiplier: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="descLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="jsonGroupBox">
       <property name="title">
        <string>Market JSON:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="jsonPlainTextEdit">
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
